proto/mls_validation/v1/service.proto

```diff
@@ -78,3 +83,28 @@ message GetAssociationStateResponse {
   xmtp.identity.associations.AssociationState association_state = 1;
   xmtp.identity.associations.AssociationStateDiff state_diff = 2;
 }
+
+// Request to validate an InboxID with the backend service. Ensures an Inbox Id <> Installation key are valid.
+message ValidateInboxIdsRequest {
+  // a single validation request
+  message ValidationRequest {
+    xmtp.identity.MlsCredential credential = 1;
```

Comment:
How does the Rust backend have access to the inbox history needed to validate the credential? That all lives in go-land?

Reply:
I think this method needs to take in all the IdentityUpdates for the inbox as well, then it can be stateless on the Rust side